src/omopcdm_stub.jl

```diff
@@ -0,0 +1,3 @@
+struct HealthTable end
```

Comment:
As I mentioned in a previous comment, could you move this into the healthtable_interface file?